operator/src/network/controller.rs

```diff
@@ -202,6 +203,13 @@ async fn reconcile(
     network: Arc<Network>,
     cx: Arc<Context<impl IpfsRpcClient, impl RngCore, impl Clock>>,
 ) -> Result<Action, Error> {
+    let meter = global::meter("keramik");
+    let runs = meter
+        .u64_counter("network_reconcile_count")
```

Comment:
Can we also get an error count? 
When the operator is unable to reconcile a network, increase a counter?